tests/system/fixtures/net-services-agent.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+if [[ "$1" == *fail* ]]
+then
+  echo "net services down"
+  #drop standard mesos traffic
+  sudo iptables -A OUTPUT -m comment --comment "remove_me" -p tcp --destination-port 5050 -j DROP
+  sudo iptables -A OUTPUT -m comment --comment "remove_me" -p tcp --destination-port 5150 -j DROP
```

Comment:
good catch.  it should be 5051 which is the slave port.

Also..  we will shortly be moving to the net partitioning I added to shakedown which is slightly different and emcompasses more.  

```
sudo iptables -F INPUT
sudo iptables -I INPUT -p tcp --dport 22 -j ACCEPT
sudo iptables -I INPUT -p icmp -j ACCEPT
sudo iptables -I OUTPUT -p tcp --sport 5051  -j REJECT
sudo iptables -A INPUT -j REJECT
```
found here: https://github.com/dcos/shakedown/blob/master/shakedown/dcos/partition_cmd